uaclient/cli.py

```diff
@@ -210,8 +210,11 @@ def action_attach(args, cfg):
         return 1
     contract_client = contract.UAContractClient(cfg)
     if not args.token:
-        bound_macaroon = sso.discharge_root_macaroon(
-            contract_client).decode('utf-8')
+        bound_macaroon_bytes = sso.discharge_root_macaroon(
+            contract_client)
+        bound_macaroon = None
```

Comment:
should this be declared above the if statement? otherwise I'm fine with this.

Reply:
Moved.